src/conn/conn_api.c

```diff
@@ -3159,6 +3159,7 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler, const char *c
     conn->page_size = __wt_get_vm_pagesize();
 
     /* Now that we know if verbose is configured, output the version. */
+    __wt_verbose_info(session, WT_VERB_RECOVERY, "%s", "opening the WiredTiger library");
     __wt_verbose(session, WT_VERB_VERSION, "%s", WIREDTIGER_VERSION_STRING);
```

Comment:
Would that make sense to combine the two?
```suggestion
    __wt_verbose_info(session, WT_VERB_RECOVERY, "opening the WiredTiger library %s", WIREDTIGER_VERSION_STRING);
```

Reply:
I thought of doing it, but the WiredTiger version always points to the version number and it has a very old date. I feel it can cause confusion to TS, so I dropped it.

Reply:
Fair enough - thank you for checking.